src/plugins/discover/public/application/main/state_management/discover_internal_state_container.ts

```diff
@@ -14,10 +14,15 @@ import {
   ReduxLikeStateContainer,
 } from '@kbn/kibana-utils-plugin/common';
 import type { DataView, DataViewListItem } from '@kbn/data-views-plugin/common';
-import type { Filter } from '@kbn/es-query';
+import type { Filter, TimeRange } from '@kbn/es-query';
 import type { DataTableRecord } from '@kbn/discover-utils/types';
 import type { UnifiedHistogramVisContext } from '@kbn/unified-histogram-plugin/public';
 
+interface InternalStateDataRequestParams {
+  timeRangeAbs?: TimeRange;
+  timeRangeRel?: TimeRange;
```

Comment:
Hi Matthias,
Still going through the changes.

One quick suggestion is to not shorten the param names:
```suggestion
  timeRangeAbsolute?: TimeRange;
  timeRangeRelative?: TimeRange;
```

Reply:
sure, will apply 👍